go/install/install_windows.go

```diff
@@ -261,9 +261,12 @@ func deprecatedStartupInfo(logFile *os.File) {
 	if appDataDir, err := libkb.AppDataDir(); err != nil {
 		logFile.WriteString("Error getting AppDataDir\n")
 	} else {
-		if exists, err := libkb.FileExists(filepath.Join(appDataDir, "Microsoft\\Windows\\Start Menu\\Programs\\Startup\\KeybaseStartup.lnk")); err == nil && exists == false {
+		autostartLinkPath := filepath.Join(appDataDir, "Microsoft\\Windows\\Start Menu\\Programs\\Startup\\KeybaseStartup.lnk")
+		if exists, err := libkb.FileExists(autostartLinkPath); err == nil && exists == false {
 			logFile.WriteString("  -- Service startup shortcut missing! --\n\n")
-		} else if err != nil {
+			return
+		}
+		if err != nil {
```

Comment:
this is different then it used to be, is it expected? this is not the `err` from `libkb.FileExists`

Reply:
fixed